### PR TITLE
Update to use new client auth model

### PR DESCRIPTION
**Pull request by lazybean**
https://docs.cord.com/reference/authentication/removing-group-from-token

- Update cord to 1.29.0;
- Do not pass group ID in client auth token;
- on page load ensure group exist and users are members
- pass group ID to the components

### CODE REVIEW

File: src/app/(cord)/layout.tsx [hunk 1]
```diff
@@ -19,21 +19,53 @@ export async function getData() {
     0,
     USERS.findIndex((user) => user.user_id === userIdCookie),
   );
-  const user = USERS[userIndex];
 
+  createAndPopulateGroup();
```
Comment (yeo-yeo) [inline src/app/(cord)/layout.tsx:23]: does this need `await`ing?

File: src/app/(cord)/layout.tsx [hunk 2]
```diff
@@ -19,21 +19,53 @@ export async function getData() {
...
+/**
+ * Creates a group and adds all users to it.
+ *
+ * In a real app, you would do this only once.
+ **/
+async function createAndPopulateGroup() {
+  const { CORD_SECRET, CORD_APP_ID } = process.env;
+  if (!CORD_SECRET || !CORD_APP_ID) {
+    console.error(
+      "Missing CORD_SECRET or CORD_ORD_ID env variable. Get it on console.cord.com and add it to .env",
```
Comment (yeo-yeo) [inline src/app/(cord)/layout.tsx:42]:
```suggestion
      "Missing CORD_SECRET or CORD_APP_ID env variable. Get it on console.cord.com and add it to .env",
```
(this was a pre-existing issue I think)

**Review by yeo-yeo — APPROVED** [review]
some comments

File: src/app/(cord)/layout.tsx (hunk 1 above)
Comment (lazybean) [reply src/app/(cord)/layout.tsx:23]: it does. Thank you!

**Pull request merged**